src/app/navbar/navbar.component.html

```diff
@@ -0,0 +1,41 @@
+<nav>
+  <div class="chefs-compass">
+    <img class="nav-logo" src="/assets/main-logo.svg" alt="nav-logo" />
+    <p class="slogan">Global flavours from your kitchen</p>
+  </div>
+  <div class="search-module">
+    <input type="text" placeholder="Dishes..." />
+    <button type="submit"><img src="/assets/mag-glass.png" /></button>
+    <div class="mini-burger">
```

Comment:
If this code is a placeholder, remove it for now cause it does not display anything